conf/conf.go

```diff
@@ -135,8 +136,18 @@ type Conf struct {
 	Settings Settings        `yaml:"settings"`
 }
 
+// Check if string is a url
 func isExternalURL(url string) bool {
-	return strings.HasPrefix(url, "http://") || strings.HasPrefix(url, "https://")
+	if _, err := netUrl.ParseRequestURI(url); err != nil {
+		return false
+	}
+
+	parts, err := netUrl.Parse(url)
+	if err != nil || parts.Host == "" || !strings.HasPrefix(parts.Scheme, "http") {
+		return false
+	}
+
+	return true
```

Comment:
I think we can add some logs here for error, info, and debug.

Reply:
hmm let me see what I can do.

Reply:
Answering my own question: yes we are initialized...

Reply:
if not, the StdOut would be the place for log.